game_engine.properties

```diff
@@ -10,3 +10,6 @@ Map_List_File = http://raw.githubusercontent.com/triplea-game/triplea/master/tri
 lobby_properties_file_url = http://raw.githubusercontent.com/triplea-game/triplea/master/lobby_server.yaml?raw=true
 lobby_properties_file_backup = lobby_server.yaml
 
+# Override setting for max memory in MB, leave blank to use default.
+# It is recommended to use the default, setting a value too high or too low can cause TripleA to be unstable.
+max_memory=1024
```

Comment:
Good catch, that was left over from testing. Updated in: 676a2f3, property is now blank to match the commentary.